### ISSUE TITLE
Why do you divide the data set like this?

### ISSUE THREAD

**Issue by Cds3851** [Hironsan/BossSensor#33]
X_train, X_test, y_train, y_test = train_test_split(images, labels, test_size=0.3, random_state=random.randint(0, 100))
    X_valid, X_test, y_valid, y_test = train_test_split(images, labels, test_size=0.5, random_state=random.randint(0, 100))

**Comment by Cds3851** [comment]
X_train, X_test, y_train, y_test = train_test_split(images, labels, test_size=0.3, random_state=random.randint(0, 100))
        X_valid, X_test, y_valid, y_test = train_test_split(images, labels, test_size=0.5, random_state=random.randint(0, 100))

**Comment by Majdi8** [comment]
إصلاح مشكله
